src/service.ts

```diff
@@ -128,6 +129,11 @@ export class Service extends Construct {
    */
   public readonly scalableTaskCount?: ecs.ScalableTaskCount;
 
+  /**
+   * The application target group.
+   */
+  public targetGroup?: ApplicationTargetGroup;
```

Comment:
We need to test that this is set (ie non-null) after adding an HTTPLoadBalancer extension.